db/version_edit.cc

```diff
@@ -648,13 +652,11 @@ std::string VersionEdit::DebugString(bool hex_key) const {
     r.append("\n  LastSeq: ");
     AppendNumberTo(&r, last_sequence_);
   }
-  for (DeletedFileSet::const_iterator iter = deleted_files_.begin();
-       iter != deleted_files_.end();
-       ++iter) {
+  for (const auto& deleted_file : deleted_files_) {
```

Comment:
use structured binding here to give `first` and `second` intuitive names directly?

Reply:
That's a C++17 feature; we generally use C++11 because we have to support fairly old compilers.